script/import_metadata

```diff
@@ -16,7 +16,7 @@ end
 
 
 puts "Loading environment..."
-require File.join(File.dirname(__FILE__), '../config/environment.rb')
+require_relative('../config/environment')
```

Comment:
This will only work if you're in some directory that is one deeper than `Rails.root`. Is that what you're going for?